doc/source/admin/special_topics/interactivetools.rst

```diff
@@ -44,6 +44,8 @@ Server-side configuration of Galaxy InteractiveTools
 
 The **galaxy.yml** file will need to be populated as seen in **config/galaxy.yml.interactivetools**.
 
+Because Galaxy interactive tools need to resolve to a wildcard DNS, this needs to be configured to work with and existing server. For users who manage their own DNS, you can set the appropriate A records to redirect *.interactivetool.yourdomain, following the same format as below. For individual users, as a convenience, using **localhost.blankenberglab.org** in place of **server address** to resolve to 127.0.0.1 on your local machine in order to allow local deployment of interactive tools.
```

Comment:
using the `blankenberglab.org` domain here seems a bit confusing

Reply:
This is meant as a temporary measure, since the server exists and is configured, until something can be made more permanent, such as interactivetools.galaxy.org. @dannon, want to chime in?

Reply:
Yeah.  It's necessary for this whole thing to work to have wildcard subdomain resolution.  localhost.blankenberglab.org is just a convenient reference for folks who want to do it.

```
[yoplait@incus:~/work/galaxy/client] dig localhost.blankenberglab.org           checkbox-bugfix

; <<>> DiG 9.10.6 <<>> localhost.blankenberglab.org
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 3743
;; flags: qr rd ra; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 4096
;; QUESTION SECTION:
;localhost.blankenberglab.org.	IN	A

;; ANSWER SECTION:
localhost.blankenberglab.org. 600 IN	A	127.0.0.1

;; Query time: 96 msec
;; SERVER: 192.168.0.1#53(192.168.0.1)
;; WHEN: Mon Apr 27 15:04:38 EDT 2020
;; MSG SIZE  rcvd: 73
```